drivers/entropy/entropy_b9x_trng.c

```diff
@@ -6,7 +6,11 @@
 
 #define DT_DRV_COMPAT telink_b9x_trng
 
+#if CONFIG_SOC_RISCV_TELINK_B91 || CONFIG_SOC_RISCV_TELINK_B92
 #include <trng.h>
+#elif CONFIG_SOC_RISCV_TELINK_B95
+#include <lib/include/trng/trng.h>
```

Comment:
Add "lib/include/trng" to CMake include list and revert this change

Reply:
will add the lib/include /trng into cmake list .